src/middle/Middle.ml

```diff
@@ -32,16 +32,16 @@ let string_of_location_span loc_sp =
     let open Format in
     let file_line_col_string =
       if bf = ef then
-        sprintf "file %s, %s" bf
+        sprintf "'%s', %s" bf
           ( if bl = el then
             sprintf "line %d, %s" bl
               ( if bc = ec then sprintf "column %d" bc
               else sprintf "columns %d-%d" bc ec )
           else sprintf "line %d, column %d to line %d, column %d" bl bc el ec
           )
       else
-        sprintf "file %s, line %d, column %d to file %s, line %d, column %d" bf
-          bl bc ef el ec
+        sprintf "'%s', line %d, column %d to file %s, line %d, column %d" bf bl
```

Comment:
probably you want to remove the file and add some quotes around the name here too

Reply:
Good idea! Does this address your other comment that "this is not the behavior we want?"

Reply:
It does!